build-system/tasks/cherry-pick.js

```diff
@@ -0,0 +1,179 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+const argv = require('minimist')(process.argv.slice(2));
+const log = require('fancy-log');
+const {getOutput} = require('../common/exec');
+const {green, cyan, red, yellow} = require('ansi-colors');
+
+const RTV_PREFIXES = {
+  '00': 'experimental',
+  '01': 'stable',
+  '02': 'control',
+  '03': 'beta',
+  '04': 'nightly',
+  '05': 'nightly-control',
+};
+
+/**
+ * Executes a shell command, and logs an error message if the command fails.
+ *
+ * @param {string} cmd
+ * @param {string} msg
+ * @return {!Object}
+ */
+function execOrLog(cmd, msg) {
+  const result = getOutput(cmd);
+  if (result.status) {
+    log(yellow('ERROR:'), msg);
+    throw new Error(result.stderr);
+  }
+
+  return result;
+}
```

Comment:
It's not similar, as `execWithError` uses `stderr` to determine if something failed, but git sometimes writes to stderr even though the process exits with code 0. I was hesitant to add it to `exec` for now because it uses a different convention for "success" (ie. the status instead of the output) and actually throws an error.

Since it's sorta customized to how this task uses it, I'm not sure if there's much to be gained by moving it to `exec`, but I don't feel too strongly about it

Reply:
I feel strongly about this :)